src/code/codeblock.scss

```diff
@@ -26,6 +26,10 @@
     }
   }
 
+  .iui-button {
+    gap: unset;
```

Comment:
Why unset?

Reply:
Because gap is not needed in this.
No gap:
![image](https://user-images.githubusercontent.com/83585998/142592721-9cb42911-1701-4935-bafe-fae4f508efc2.png)
Gap: 
![image](https://user-images.githubusercontent.com/83585998/142592904-dc71baaf-7a2d-48da-a188-2bde7ea6fe20.png)